src/ShopBundle/objects/db/TShopOrder.class.php

```diff
@@ -540,6 +543,32 @@ protected function SaveOrderNotificationDataToOrder($bOrderSend, $oMail, $sSendT
         $this->AllowEditByAll(false);
     }
 
+    protected function SetSendOrderNotificationState(bool $orderSend): void
```

Comment:
Naming "orderSend": Does this mean the order is sent (> orderSent) or that it will be sent (>sendOrder)?